### ISSUE TITLE
SA1125 UseShorthandForNullableTypes incorrectly reported in XML comment

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#385]
SA1125 is getting reported for the following XML documentation comment:

```csharp
/// <seealso cref="Nullable{T}"/>
```
